apimatic_core/api_call.py

```diff
@@ -1,26 +1,25 @@
 from apimatic_core.configurations.endpoint_configuration import EndpointConfiguration
 from apimatic_core.configurations.global_configuration import GlobalConfiguration
-from apimatic_core.logger.endpoint_logger import EndpointLogger
+from apimatic_core.logger.sdk_logger import LoggerFactory
 from apimatic_core.response_handler import ResponseHandler
 
 
 class ApiCall:
 
     @property
     def new_builder(self):
-        return ApiCall(self._global_configuration, logger=self._endpoint_logger.logger)
+        return ApiCall(self._global_configuration)
 
     def __init__(
             self,
-            global_configuration=GlobalConfiguration(),
-            logger=None
+            global_configuration=GlobalConfiguration()
     ):
         self._global_configuration = global_configuration
         self._request_builder = None
         self._response_handler = ResponseHandler()
         self._endpoint_configuration = EndpointConfiguration()
-        self._endpoint_logger = EndpointLogger(logger)
-        self._endpoint_name_for_logging = None
+        self._sdk_configuration = self._global_configuration.get_http_client_configuration()
```

Comment:
do we need to create a separate variable sdk_configuration as its being  used in the constructor only not elsewhere

Reply:
Done